index.html

```diff
@@ -124,16 +125,9 @@ <h4>EditContext state</h4>
                 <li><dfn>underline style</dfn>, a {{UnderlineStyle}} which is the preferred underline style of the decorated [=text=] range.</li>
                 <li><dfn>underline thickness</dfn>, a {{UnderlineThickness}} which is the preferred underline thickness of the decorated [=text=] range.</li>
             </ul>
-            <p><dfn>codepoint location</dfn> is a struct that captures the location (in [=client coordinate=]s) of codepoints from [=text=].  Each struct represents a contigious run of codepoints starting at an offset within [=text=] indicated by [=start index=].  The struct contains:</p>
-            <ul>
-                <li><dfn>start index</dfn> which is an offset into [=text=] that respresents the position before the first codepoint whose location is reported by the first member of [=codepoint rects=] array.</li>
-                <li><dfn>codepoint rects</dfn> is an array of {{DOMRect}} defining the bounding box of each codepoint.</li>
-            </ul>
             <p class="note">
-                [=Codepoint locations=] provides the means for the user agent to query multiple ranges of [=text=] for positioning information.
-                Different platforms may require different positions to be cached to fulfill queries from the [=Text Input Service=] and this API shape allows for that flexibility.
-                It would be good if the API could be simplified to require just the positions of the actively composed text along with the [=selection bounds=] and [=control bounds=].
-                Exploration is needed to see if that is possible.
+              [=Codepoint rects=] provides the means for the user agent to query a range of [=text=] for positioning information.
+              Different platforms may require different positions to be cached to fulfill queries from the [=Text Input Service=].
```

Comment:
>  Different platforms may require different positions to be cached to fulfill queries from the [=Text Input Service=].

Perhaps a note with some examples would be helpful to understand why there is a difference in how platforms use these positions? (Also, can be in a separate PR).